src/components/LandingHowItWorks/index.tsx

```diff
@@ -168,16 +168,40 @@ export const LandingHowItWorks: FC = () => (
         "py-8 sm:py-12 md:py-16",
       ].join(" ")}
     >
+      <aside
+        className={[
+          "p-8 mx-auto mb-16 md:mb-24 max-w-prose",
+          "border border-green shadow-green",
+        ].join(" ")}
+      >
+        <img
+          src='/images/illustrations/microcontroller-2.svg'
+          alt='Microcontroller'
+          className='inline-block mr-4 align-top'
+        />
+        <p className='mt-3'>
+          In Berlin gibt es ein offenes LoRaWan-Netzwerk, welches seit 2017
+          durch die TheThingsNetwork (TTN) Community betrieben wird. Jeder kann
+          und darf es komplett kostenlos benutzten, seine Sensoren mit dem
+          Netzwerk verbinden und die Stadt auf eigene Faust vermessen.
+        </p>
+      </aside>
       <h1
         className={[
           "text-xl sm:text-2xl md:text-3xl",
           "text-purple font-bold font-headline",
           "mt-6 mb-1",
         ].join(" ")}
       >
-        Und so geht’s:
+        Stadtpuls unterstützt die Anbindung an TTN!
```

Comment:
It would be nice to have a label saying "More coming!" next to the title

Reply:
good point --> will add it